tests/behat/features/api/get_scratch_remix_programs.feature

```diff
@@ -0,0 +1,125 @@
+      | 1  | program 1 | p1          | Catrobat | 3         | 12    | 01.01.2013 12:00 | 0.8.5   | true       |
+      | 2  | program 2 |             | Catrobat | 33        | 9     | 01.02.2013 13:00 | 0.8.5   | false      |
+      | 3  | program 3 |             | Catrobat | 132       | 33    | 01.01.2012 13:00 | 0.8.5   | false      |
+      | 4  | program 4 | p1          | Catrobat | 5         | 13    | 01.01.2013 12:00 | 0.8.5   | true       |
+      | 5  | program 5 |             | Catrobat | 36        | 7     | 01.02.2013 13:00 | 0.8.5   | false      |
+      | 6  | program 6 |             | Catrobat | 135       | 31    | 01.01.2012 13:00 | 0.8.5   | false      |
+      | 7  | program 7 |             | Catrobat | 133       | 35    | 01.01.2012 13:00 | 0.8.5   | false      |
+      | 8  | program 8 |             | Catrobat | 134       | 30    | 01.01.2012 13:00 | 0.8.5   | false      |
+      | 9  | program 9 |             | Catrobat | 131       | 38    | 01.01.2012 13:00 | 0.8.5   | false      |
+      | 12  | program 12 |             | Catrobat | 131       | 38    | 01.01.2012 13:00 | 0.8.5   | false      |
+      | 13  | program 13 |             | Catrobat | 131       | 38    | 01.01.2012 13:00 | 0.8.5   | false      |
+
+
+
+    Given there are Scratch remix relations:
+      | scratch_parent_id | catrobat_child_id |
+      | 70058680          | 6                 |
+      | 70058680          | 5                 |
+      | 70058680          | 7                 |
+      | 70058680          | 8                 |
+      | 70058680          | 9                 |
+
+
+  Scenario: show most downloaded programs with limit and offset
+    Given I have a parameter "limit" with value "5"
+    And I have a parameter "offset" with value "0"
+    When I GET "/app/api/projects/scratchRemixes.json" with these parameters
+    Then I should get 5 programs in the following order:
+      | Name      |
+      | program 5 |
+      | program 6 |
+      | program 7 |
+      | program 8 |
+      | program 9 |
+
+  Scenario: program upload with parent-URL referring only to Scratch programs and
+  Catrobat language version 0.999 should correctly add remix relations
+    #-------------------------------------------------------------------------------------------------------------------
+    # this is how the remix graph should look like after the program is uploaded:
+    #
+    #       (SCRATCH #1)  (SCRATCH #2)
+    #                \      /
+    #                \     /
+    #                 (10)            <-- to be added (uploaded program will get ID "10")
+    #
+    #-------------------------------------------------------------------------------------------------------------------
+    Given I have a program with "catrobatLanguageVersion" set to "0.999" and "url" set to "Music Inventor [https://scratch.mit.edu/projects/29495624], The Colour Divide - Trailer [https://scratch.mit.edu/projects/70058680/]"
```

Comment:
would look cleaner if you also set the project name

Reply:
Fixed